package.json

```diff
@@ -26,10 +26,11 @@
     "@atb-as/theme": "^6.0.0-alpha.7",
     "@bugsnag/react-native": "^7.12.0",
     "@bugsnag/source-maps": "^2.3.1",
+    "@entur/abt-mobile-client-sdk": "^1.0.5-rc.0",
+    "@entur/abt-token-server-node-lib": "^1.0.0",
```

Comment:
Is this included for the types only? 🤔

Reply:
Yeah. Is going to get it's own types-lib, but not done yet by Entur. Can probably move it to dev dependencies?

Reply:
Yep. Has probably no implications for this tho, but technically "more correct"